python/lsst/source/injection/utils/make_injection_pipeline.py

```diff
@@ -155,27 +155,17 @@ def make_injection_pipeline(
     # Remove all tasks which are not to be included in the injection pipeline.
     if isinstance(excluded_tasks, str):
         excluded_tasks = set(excluded_tasks.split(","))
-    not_excluded_tasks = set()
-    for task_label in excluded_tasks:
-        # First remove tasks from their host subsets, if present.
-        try:
-            host_subsets = pipeline.findSubsetsWithLabel(task_label)
-        except ValueError:
-            pass
-        else:
-            for host_subset in host_subsets:
-                pipeline.removeLabelFromSubset(host_subset, task_label)
-        # Then remove the task from the pipeline.
-        try:
-            pipeline.removeTask(task_label)
-        except KeyError:
-            not_excluded_tasks.add(task_label)
-    if len(not_excluded_tasks) > 0:
-        grammar = "Task" if len(not_excluded_tasks) == 1 else "Tasks"
+    all_tasks = {taskDef.label for taskDef in pipeline.toExpandedPipeline()}
```

Comment:
Does this need to be `toExpandedPipeline`? (A question for @natelust)

Reply:
For context: here we're just looping over all tasks in the pipeline to get a set of task names. No actual task/config manipulation is performed in this block.

Reply:
It's never quite clear for me when I need to expand my pipeline, and when I can get away with using the base pipeline object. I.e., I want to be sure that I'm including all the tasks from imported pipeline YAMLs as well.